### DISCUSSION TITLE
How do I change the port?

### DISCUSSION THREAD

**Discussion by p-baum** [hoppscotch/hoppscotch#2251] [Q&A] [answered]
How do I change the port from 3000 please?  Vite.js is serving on this port.

**Comment by liyasthomas** [comment, marked as answer]
There're multiple ways to change the default port. Follow the Nuxt.js documentation on how to edit host and port here: https://nuxtjs.org/docs/features/configuration/#edit-host-and-port

The easiest method is to set `HOST` and `PORT` environment variables in `pnpm run dev` command.
ex: `HOST=0 PORT=8000 pnpm run dev`
